### ISSUE TITLE
Mobile wireframes

### ISSUE THREAD

**Issue by tenhobi** [tenhobi/flashcards#28]
Depends on #14.

Create wireframes for mobile app based on analysis of requirtments and description of use cases.

* [ ] wireframes
* [ ] wiki description

**Comment by tenhobi** [comment]
We could use something like https://wireframe.cc/, https://free.lstore.graphics/uxflow/, ...

**Comment by tenhobi** [comment]
Some drafts. We should do it better, I don't have a good app for it.

![screenshot-2018-3-4 ghj 2](https://user-images.githubusercontent.com/5287596/36949864-1aea1ffc-1fee-11e8-8931-a7f0ee8727ad.png)
![screenshot-2018-3-4 ghj](https://user-images.githubusercontent.com/5287596/36949865-1b040656-1fee-11e8-8a1f-c5951b5ee1c3.png)
![screenshot-2018-3-4 ghj 1](https://user-images.githubusercontent.com/5287596/36949866-1b1cb2a0-1fee-11e8-820f-8e6da655f3d1.png)

**Comment by tenhobi** [comment]
Hmm, maybe the app navigation should be at the bottom of the app, in Instagram-alike style and it will contain:

1. home (currently working-on courses & my courses—separated by tabs on the top of the page)
1. search (both for courses and users)
1. my profile
1. settings??

And in drawer (the thing which appears when you swipe from the left corner of the app) there will be some about, links, contacts, copyright etc.

**Comment by tenhobi** [comment]
Today I will try to do some wireframes on paper, just to imagine what would be the best layout etc. for our app.

**Comment by tenhobi** [comment]
Today I will try to do some wireframes on paper, just to imagine what would be the best layout etc. for our app.

**Comment by vojtechdalecky** [comment]
I am going to join you today. Call at 21:00?

**Comment by tenhobi** [comment]
OK.

**Comment by vojtechdalecky** [comment]
I downloaded some (iPhone X) mockups and now i am working on graphic implementation of our ideas from yesterday. 👨🏻‍💻

**Comment by tenhobi** [comment]
@vojtechdalecky you should probably do some nice graphics for the wireframes first, so we can add it to our analysis, which is critical for our 1st iteration and we have to finish it first. :) 

The design is not necessary, it's only nice-to-have for 2nd iteration. :)

**Comment by vojtechdalecky** [comment]
Okey, i will prefer the wireframes (I can to project them in mockups too anyway). ✌️
